docs/quickstart.md

```diff
@@ -37,6 +37,11 @@ python3 -m venv .venv
 source .venv/bin/activate
 ```
 
+:::info
```

Comment:
Can we instead do this with tabs? Like in some of the cross-platform examples below?  Looks much cleaner in my opinion.